js/script.js

```diff
@@ -96,3 +96,16 @@ function displayMyVote(){
     }
   }
 }
+
+let clearForm = () => {
+  document.getElementById("contest-form").reset();
+}
+let conSubmit = () => {
+  let fName = document.getElementById("fN").value;
+  let lName = document.getElementById("lN").value;
+  let party = document.getElementById("pty").value;
+  let office = document.getElementById("of").value;
+  let manifesto = document.getElementById("man").value;
```

Comment:
I think your `ids` should be descriptive. Please use descriptive names in your code. Thanks

Reply:
Okay, I will make the changes